### PR TITLE
populate include and lib paths via llvm-config

### PR DESCRIPTION
**Pull request by gurinderu**

### CODE REVIEW

File: sbtplugin/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala [hunk 1]
```diff
@@ -52,120 +53,124 @@ object ScalaNativePluginInternal {
...
+        val links = compileNir(opts).map(_.name)
+        compileLl(clang, target, appll, binary, links, linkage, clangOpts)
+
+        binary
+      },
+      run := {
+        val binary = nativeLink.value
+
+        Process(abs(binary)).!
+      }
```
Comment (densh) [inline sbtplugin/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala]: This block of changes seems to have too big of a diff for no good reason. Please avoid changing the parts of the codebase that are not related to the main topic of the pr. For example removing blank lines between settings was not necessary.

File: sbtplugin/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala [hunk 2]
```diff
@@ -85,16 +85,15 @@ object ScalaNativePluginInternal {
...
-                Process("llvm-config --includedir").lines_!).getOrElse(
-                Stream.empty)).map(s => s"-L$s")
-      }
-
-        nativeEmitDependencyGraphPath := None,
+        val includes = ("/usr/local/include" #:: Try(
+                Process("llvm-config --includedir").lines_!)
+              .getOrElse(Stream.empty)).map(s => s"-I$s")
+        val libs = ("/usr/local/lib" #:: Try(
+                Process("llvm-config --includedir").lines_!)
```
Comment (wookietreiber) [inline sbtplugin/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala]: This should be `llvm-config --libdir`. Probably a copy-paster.

**Comment by densh** [comment]
Please squash in one commit but otherwise it's LGTM!

**Comment by densh** [comment]
LGTM!
